backend/src/routes/api/cluster-settings/clusterSettingsUtils.ts

```diff
@@ -23,28 +24,32 @@ export const updateClusterSettings = async (
 ): Promise<{ success: boolean; error: string }> => {
   const coreV1Api = fastify.kube.coreV1Api;
   const namespace = fastify.kube.namespace;
-  const query = request.query as { [key: string]: string };
+  const body = request.body as { [key: string]: any };
```

Comment:
I would avoid using any in TypeScript, we want to strictly type as much as possible.

Reply:
done.